### PR TITLE
Add ABOUT.md to xrust docs

### PR DESCRIPTION
**Pull request by hjr3**

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Rust is a systems programming language focused on three goals: safety, speed, and concurrency. Rust does not use a garbage collector, but instead ensures safety and optimizes code at compile time. The concept of _ownership_ is how Rust achieves its largest goal, memory safety.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Is there a good blog post or something about ownership? I can't see what it might mean without a bit of extra context.

**Comment by EduardoBautista** [comment]
This looks good, thanks!

File: docs/ABOUT.md (hunk 1 above)
Comment (EduardoBautista) [inline docs/ABOUT.md]: Oops, merged before reading this.  This is explained in "The Book".  Maybe adding a link to that chapter would be helpful.

Comment (hjr3) [inline docs/ABOUT.md]: I will add a link to https://doc.rust-lang.org/book/ownership.html

Comment (hjr3) [inline docs/ABOUT.md]: Opened https://github.com/exercism/xrust/pull/70

**Pull request merged**